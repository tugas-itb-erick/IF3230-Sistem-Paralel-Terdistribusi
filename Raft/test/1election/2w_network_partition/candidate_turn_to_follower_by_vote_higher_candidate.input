5
set_timeout 1 0
simulate 10
disconnect 1 3
disconnect 1 4
disconnect 1 5
disconnect 2 3
disconnect 2 4
disconnect 2 5
set_timeout 3 0
simulate 10
get_role 1
get_role 2
get_role 3
get_role 4
get_role 5
simulate 100
get_role 1
get_role 2
get_role 3
get_role 4
get_role 5
set_timeout 3 1000
set_timeout 4 1000
set_timeout 5 1000
set_timeout 1 5
set_timeout 2 0
simulate 10
set_timeout 1 0
set_timeout 2 5
simulate 2
set_timeout 1 5
set_timeout 2 0
simulate 2
set_timeout 1 0
set_timeout 2 5
simulate 2
set_timeout 1 5
set_timeout 2 0
simulate 2
set_timeout 1 0
set_timeout 2 5
simulate 2
get_state 1
get_state 2
connect 1 3
connect 1 4
connect 1 5
connect 2 3
connect 2 4
connect 2 5
set_timeout 1 1000
set_timeout 2 0
set_timeout 3 1000
set_timeout 4 0
set_timeout 5 1000
timestep
get_state 4
flush_outbox
get_state 4
end_simulation