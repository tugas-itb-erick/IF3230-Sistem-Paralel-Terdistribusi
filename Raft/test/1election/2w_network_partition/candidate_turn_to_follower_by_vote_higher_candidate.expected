leader
follower
leader
follower
follower
leader
follower
leader
follower
follower
S1 candidate term:9 voted_for:1 commit_index:0 data:0 logs:T0[0]
S2 follower term:9 voted_for:1 commit_index:0 data:0 logs:T0[0]
S4 candidate term:4 voted_for:4 commit_index:0 data:0 logs:T0[0]
S4 follower term:10 voted_for:2 commit_index:0 data:0 logs:T0[0]
